ai_genomics/pipeline/patent_data/get_ai_genomics_patents.py

```diff
@@ -0,0 +1,146 @@
+        "abstract_localized.text as abstract_text, abstract_localized.language as abstract_language, "
+        "publication_date, filing_date, grant_date, priority_date, inventor, assignee, entity_status "
+    )
+
+    genomics_ai_q = (
+        f"WITH "
+        f"genomics_ids AS ({genomics_q}) "
+        "SELECT "
+        f"{genonmics_ai_fields}"
+        "FROM `patents-public-data.patents.publications`, UNNEST(cpc) AS cpc, UNNEST(ipc) AS ipc, "
+        "UNNEST(title_localized) AS title_localized, UNNEST(abstract_localized) AS abstract_localized "
+        "INNER JOIN genomics_ids USING(publication_number) "
+        f"WHERE cpc.code in ({cpc_ai_ids}) OR ipc.code in ({ipc_ai_ids});"
+    )
+
+    return genomics_ai_q
+
+
+def select_unique_ai_genomics_patents(
+    full_table_name: str,
+) -> str:
+    """Selects unique ai-genomics patents based on publication_number.
+
+    Args:
+        full_table_name: name of table to query.
+    """
+    unique_ai_genomics_patents = (
+        "SELECT * FROM ("
+        "SELECT *, ROW_NUMBER() OVER (PARTITION BY publication_number) row_number "
+        f"FROM `{full_table_name}`) "
+        "WHERE row_number = 1"
+    )
+
+    return unique_ai_genomics_patents
+
+
+if __name__ == "__main__":
+
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        "--table_name",
+        help="the name of the table to pull ai genomics patents from",
+        default="ai_genomics",
+    )
+
+    args = parser.parse_args()
+    table_name = args.table_name
+    full_table_name = DATASET_NAME + "." + table_name
+
+    conn = est_conn()
+    tables = conn.list_tables(DATASET_NAME)
+    table_names = [
+        "{}.{}.{}".format(table.project, table.dataset_id, table.table_id)
+        for table in tables
+    ]
+    unique_ai_genomics_patents_q = select_unique_ai_genomics_patents(
+        full_table_name=full_table_name
+    )
+
+    if full_table_name not in table_names:
+        try:
+            ai_genomics_table_q = genomics_ai_query()
+
+            job_config = bigquery.QueryJobConfig(destination=full_table_name)
+            query_job = conn.query(ai_genomics_table_q, job_config=job_config)
+            query_job.result()
+            logger.info("Query results loaded to the table {}".format(full_table_name))
+        except Forbidden:
+            raise Error(f"Time out error. Try again in 2-3 hours.")
+
+    try:
+        genomics_ai_df = conn.query(unique_ai_genomics_patents_q).to_dataframe()
+        # subset for only english abstracts and drop row_number
+        genomics_ai_df = (
+            genomics_ai_df[genomics_ai_df["abstract_language"] == "en"]
+            .drop(columns="row_number")
+            .pipe(replace_missing_values_with_nans)
+            .pipe(convert_date_columns_to_datetime)
+        )  #
+        # save to s3
+        save_to_s3(bucket_name, genomics_ai_df, S3_SAVE_FILENAME)
+    except Forbidden:
+        raise Error(f"Time out error. Try again in 2-3 hours.")
```

Comment:
The logging exception could be used here instead and then we don't need the `error_utils.py` file

```suggestion
    except Forbidden:
        logging.exception("Time out error. Try again in 2-3 hours.")
```